tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -1,11 +1,3 @@
-
-# this is repeated from the core E+ build, which is unfortunate
-# we need to either:
-# - figure out how to build the unit tests without the Python dependency, which would mean putting a conditional DEFINE in the PluginManager.hh code, or
-# - just get this once at highest level configure time and save it for everywhere
-find_package(PythonLibs 3 REQUIRED)
```

Comment:
We won't be trying to link to Python on all cases here, so this is moved to a lower block.